packages/@lwc/integration-karma/test/rendering/if-inside-custom-render/index.spec.js

```diff
@@ -0,0 +1,14 @@
+import { createElement } from 'lwc';
+import CustomRender from 'x/customRender';
+
+describe('using custom renderer with lwc:if', () => {
+    it('should just work test debug', async function () {
```

Comment:
Did you mean to name `it` this way? :)

Reply:
I thought I changed this....that's embarrassing 😓

Reply:
We've all been there. I like `yolo haha` myself